docs/source/configuration/conditions.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Conditions
+subtitle: Set conditions for when events or instruments are triggered
+description: 
+---
+
+<EnterpriseFeature />
+
+Conditions set when an [instrument](./instruments) is mutated or an [event](./events) is triggered.
+
+Here is an example of a condition on a custom instrument:
+
+```yaml title="router.yaml"
+telemetry:
+  instruments:
+    router:
+      my.instrument:
+        # ...
+        # This instrument will only be mutated if the condition evaluates to true
+        condition:
+          all:
+            - any:
+                - eq:
+                    - "val"
+                    - request_header: x-req-header
+            - eq:
+                - "foo"
+                - response_header: x-resp-header
+```
+
+### Condition configuration
+
+The available basic conditions:
+
+- [`eq`](#eq)
+- [`not`](#not)
+- [`all`](#all)
+- [`any`](#any)
+
+You can create complex conditions by using these basic conditions as building blocks.
+
+#### `eq`
+
+The `eq` condition is an equality test between [selectors](./selectors) or values.
+
+For example, the following condition checks the value of `x-req-header` is equal to `val`:
+
+```yaml
+eq:
+  - "val"
+  - request_header: x-req-header
+```
+
+You can use selectors on both sides of the equality test:
+
+```yaml
+eq:
+  - request_header: x-req-header1
+  - request_header: x-req-header2
+```
+
+Values may be of types `string`, `number` or `boolean`.
+
+#### `not`
+
+The `not` condition is a negation of the nested condition.
+
+For example, the following condition checks the value of `x-req-header` is _not_ equal to `val1`:
+
+```yaml
+not:
+  eq:
+    - "val1"
+    - request_header: x-req-header2
+```
+
+#### `all`
+
+The `all` condition is a list of conditions that all must be true in order for the condition to be true.
+
+For example, the following `all` condition has a list of `eq` conditions that check the values of `x-req-header1` and `x-req-header2`, and both `eq` conditions must be true in order for the `all` condition to be true:
+
+```yaml
+all:
+  - eq:
+      - "val1"
+      - request_header: x-req-header1
+  - eq:
+      - "val2"
+      - request_header: x-req-header2
+```
+
+#### `any`
+
+The `any` condition is a list of conditions of which at least one must be true for the condition to be true.
+
+For example, the following `any` condition has a list of `eq` conditions that check the values of `x-req-header1` and `x-req-header2`, and at least one of the `eq` conditions must be true in order for the `all` condition to be true:
+
+```yaml
+any:
+  - eq:
+      - "val2"
+      - request_header: x-req-header1
+  - eq:
+      - "val2"
+      - request_header: x-req-header2
+```
+
+### Configuration reference
```

Comment:
I would replace lines 35 - 38 with this table to give more information upfront and remove it from the end.